client/cloud_credentials.go

```diff
@@ -42,17 +42,17 @@ type AzureCredentialsValuePayload struct {
 }
 
 type AwsCredentialsCreatePayload struct {
-	Name           string                     `json:"name"`
-	OrganizationId string                     `json:"organizationId"`
+	Name           string                     `json:"name,omitempty"`
```

Comment:
what are those `omitempty` here for?

Reply:
This prevent sending in the json something like:
organizationId: "" and name: "".

The API may have ignored it, but it's still incorrect.